nussl/ml/networks/separation_model.py

```diff
@@ -190,10 +212,6 @@ def forward(self, data):
                     for k in stats[o]: 
                         stats_desc += f"\n\t\t\t{k}: {stats[o][k]:.4f}"
 
-                min_max = ", ".join(
```

Comment:
Did you mean to remove this? What did this even do? Was this unused from a verbose mode or something?